client/src/components/feedback/comment.js

```diff
@@ -29,18 +35,61 @@ class FeedbackComment extends Component {
     }
   }
 
+  likeFeedback() {
+    Api.post(`feedback/like?feedbackId=${this.props.item.id}&personId=${Person.userInfo.id}`)
+      .then((response) => {
+        if (response.status === 201) {
+          if (this.state.item.id === response.data.id) {
+            this.setState((prevState) => {
+              const item = { ...prevState.item };
+              item.likes = response.data.likes;
+              return { item };
+            });
+          }
+        } else {
+          setTimeout(() => {
+            this.setState({
+              unexpectedError: false,
+            });
+          }, 3000);
```

Comment:
Could you remove all timeouts?
In 2020 we have very fast devices and all developers and services and companies try to make applications faster but you add additional complications in code and UX. Very bad idea

Reply:
This timeout is for notification show. We can't show static notification. In some period after show this should be hidden. That is what timeout does here.

Reply:
React bootstrap has highlevel handlers for similar cases https://react-bootstrap.github.io/components/toasts/#autohide
https://react-bootstrap.github.io/components/alerts/#api - onClose() method
You should use alert or toast for messages, not bages

Reply:
I have discovered information by this links. So, I want to say that due to my current implementaton badge (or text if we refuse using badge) is shown immediately near like/unlike button and doesn't change whole feedback card views. Both toast and alert will push the rest part of page to be situated when shown. 

I see multiple desicions.
1. We don't use timeout, but then we can show notification till user moves to another page.
2. Don't use any notification, but hover message as it is right now.

All in all, I believe there is a better way to implement this, even with react component lifecycle methods - I mean stop rendering component when it is inactive. But, due to the fact we are not UI developers, I'd leave it like it is right now till our memory leak issue can appear as potential bug.